addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -252,6 +260,7 @@ def _l10n_es_edi_facturae_inv_lines_to_items(self, conversion_rate=None):
             totals['total_taxes_withheld'] += sum((abs(tax["tax_amount"]) for tax in taxes_withheld_computed))
 
             invoice_line_values.update({
+                'ReceiverTransactionReference': self._l10n_es_edi_facturae_get_receiver_transaction_reference(line),
```

Comment:
Not sure having a dedicated method for this is necessary :thinking: 
Couldn't we simply use a variable?

Reply:
I did this initially then decided to go with a method instead. Felt it was better not to add more lines to an already  big method but I can revert it if you'd like!

Reply:
I think having a method is useful if it needs to be overridden or if the same logic is repeated several times, but in this case, I think a simple variable would be enough :blush: